reporter/reporter.go

```diff
@@ -73,6 +82,15 @@ func Report(ctx context.Context, err error) error {
 	return reportWithLevel(ctx, DefaultLevel, e)
 }
 
+// Flush the Reporter embedded within the context.Context
+func Flush(ctx context.Context) {
+	if r, ok := FromContext(ctx); ok {
+		r.Flush()
+	}
+
+	panic("No reporter in provided context.")
```

Comment:
This is not necessary. Actually this would trigger every time. But if a reporter is not found in the context to Flush, I think its safe to ignore that.